src/js/vic-v2/helpers.jsx

```diff
@@ -163,3 +163,29 @@ export function submit(form, formConfig) {
   });
 }
 
+export function hasSavedForm(savedForms, formID) {
+  return savedForms.some(({ form }) => form === formID);
+}
+
+export function scrollToElement(selectorOrElement) {
```

Comment:
We've been using react-scroll for this. If you search for `scrollToTop` in the codebase you can find examples of how it's used.

Reply:
Thanks for pointing that out, will update.